Frameworks/UIKit/UIView.mm

```diff
@@ -530,17 +543,18 @@ - (void)_initPrivWithFrame:(CGRect)frame xamlElement:(WXFrameworkElement*)xamlEl
     viewCount++;
 
     // If we don't have a backing xamlElement, create one for this UIView type
-    if (!xamlElement) {
+    if (xamlElement == nil) {
         xamlElement = [[self class] createXamlElement];
     }
 
     // Create and initialize our backing presentation layer
-    if (!xamlElement) {
+    if (xamlElement == nil) {
```

Comment:
No good reason. At one point I had changed the xamlElement to a C++/WinRT type, which is why I did this, but then I changed it again to RTObject*.

---
In reply to: [109245834](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109245834) [](ancestors = 109245834)